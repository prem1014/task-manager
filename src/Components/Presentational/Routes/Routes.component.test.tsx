import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import AppRoutes from './Routes.component';

describe('AppRoutes component test suite', () => {
    Enzyme.configure({ adapter: new Adapter() });
    let componentWrapper: any;

    beforeEach( () => {
        componentWrapper = shallow(<AppRoutes />);
    });

    it('AppRoutes component snapshot test', () => {
        expect(componentWrapper).toMatchSnapshot();
    });
});