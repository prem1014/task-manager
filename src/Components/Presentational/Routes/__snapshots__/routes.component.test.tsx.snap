// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRoutes component test suite AppRoutes component snapshot test 1`] = `ShallowWrapper {}`;
